### PR TITLE
Add gh actions

### PR DESCRIPTION
**Pull request by keflavich**
I'd like to get tests running again.  I copied actions from spectral-cube.  I hope those are right.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            python-version: 3.8
+            name: Python 3.8 with minimal dependencies
+            toxenv: py38-test
+          - os: ubuntu-latest
+            python-version: 3.7
+            name: Python 3.7 with minimal dependencies
+            toxenv: py37-test
+          - os: ubuntu-latest
+            python-version: 3.9
+            name: Python 3.9 with all non-visualization dependencies (except CASA)
```
Comment (e-koch) [inline .github/workflows/main.yml:31]: Drop w/ + w/o CASA run

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      matrix:
+        include:
+          - os: ubuntu-latest
+            python-version: 3.9
+            name: Python 3.9 with minimal dependencies
+            toxenv: py39-test
+          - os: ubuntu-latest
+            python-version: 3.9
+            name: Python 3.9 with no visualization + coverage
+            toxenv: py39-test-novis-cov
```
Comment (e-koch) [inline .github/workflows/main.yml:20]: `-novis` isn't defined in the tox file.

https://github.com/radio-astro-tools/pvextractor/blob/master/tox.ini#L3

py39 also needs to be added there.

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            python-version: 3.7
+            name: Python 3.7 with minimal dependencies
+            toxenv: py37-test
+          - os: ubuntu-latest
+            python-version: 3.9
+            name: Python 3.9 with all non-visualization dependencies (except CASA)
+            toxenv: py39-test-novis
+          - os: ubuntu-18.04
+            python-version: 3.6
+            name: Python 3.6 with minimal dependencies and CASA
```
Comment (e-koch) [inline .github/workflows/main.yml:35]: Drop w/ + w/o CASA run

**Review by e-koch — CHANGES_REQUESTED** [review]
The tox file needs to be updated in a few places.

Also can remove the travis.yml file.

**Comment by keflavich** [comment]
@e-koch does this look good now?  Do we need to activate actions...

**Comment by e-koch** [comment]
@keflavich I _think_ so.

I can't remember if actions should just start or not. Since this PR only affects CI, I'm in favour of merging and fixing CI issues in another PR once they're running.

**Review by e-koch — APPROVED** [review]

**Comment by keflavich** [comment]
it's green on my side!

**Pull request merged**